### PR TITLE
last update

### PR DESCRIPTION
**Pull request by JoumanaE**

### CODE REVIEW

File: app/tuition_blueprint/react/package.json [hunk 1]
```diff
@@ -15,8 +15,8 @@
     "react-autosuggest": "9.3.2",
     "react-breadcrumbs": "^1.6.6",
     "react-breadcrumbs-dynamic": "^1.0.13",
-    "react-dom": "16.0.0",
-    "react-number-format": "^2.0.4",
+    "react-dom": "^16.0.0",
+    "react-number-format": "^3.3.0",
     "react-redux": "^5.0.6",
     "react-router-breadcrumbs": "^2.1.1",
     "react-router-dom": "^4.2.2",
```
Comment (btalberg) [inline app/tuition_blueprint/react/package.json]: It is probably worth updating to the latest material-ui releases before we roll this to production. I took a look at the most recent releases:
```
    "material-ui": "1.0.0-beta.38",
    "material-ui-icons": "1.0.0-beta.36",
    "material-ui-pickers": "v1.0.0-rc.3",
```

File: app/tuition_blueprint/react/src/components/Customers.jsx [hunk 2]
```diff
@@ -75,7 +75,9 @@ class RecurringPayment extends Component {
   render() {
     if (this.props.recurringPayment && this.props.items.length >= 1) {
       var item = this.props.items.filter(item => item.id === this.props.recurringPayment.item_id)[0]
-      var endDate = new Date(this.props.recurringPayment.end_date);
+      // debug the length
```
Comment (btalberg) [inline app/tuition_blueprint/react/src/components/Customers.jsx]: ^^^ A left over debug comment

**Comment by JoumanaE** [comment]
Added all of the updates

File: app/tuition_blueprint/react/src/components/RecurringPaymentAddForm.jsx [hunk 3]
```diff
@@ -122,11 +122,7 @@ class RecurringPaymentAddForm extends Component {
 
 validatePayment = () => {
   for(var i=0; i< this.componentsToValidate.length; i++) {
-    if(this.componentsToValidate[i].props.name === "checkingAccountNumber"){
-      if(this.componentsToValidate[i].props.valid === "false"){
-        return false;
-      }
-    }
+    return false;
```
Comment (btalberg) [inline app/tuition_blueprint/react/src/components/RecurringPaymentAddForm.jsx]: Ah, I see why this wasn't working. You still need to run the validation test:

```
if(this.componentsToValidate[i].props.valid === "false"){	
     return false;	
}
```

The change here is just taking out the conditional that only checks the validity of the "checkingAccountNumber".

**Pull request merged**